Backpack/src/main/java/net/skyscanner/backpack/util/ExperimentalBackpackApi.kt

```diff
@@ -0,0 +1,21 @@
+package net.skyscanner.backpack.util
+
+@Suppress("DEPRECATION")
+@Experimental(level = Experimental.Level.ERROR)
+@RequiresOptIn(level = RequiresOptIn.Level.ERROR)
+@Retention(AnnotationRetention.BINARY)
+@Target(
+  AnnotationTarget.CLASS,
+  AnnotationTarget.ANNOTATION_CLASS,
+  AnnotationTarget.PROPERTY,
+  AnnotationTarget.FIELD,
+  AnnotationTarget.LOCAL_VARIABLE,
+  AnnotationTarget.VALUE_PARAMETER,
+  AnnotationTarget.CONSTRUCTOR,
+  AnnotationTarget.FUNCTION,
+  AnnotationTarget.PROPERTY_GETTER,
+  AnnotationTarget.PROPERTY_SETTER,
+  AnnotationTarget.TYPEALIAS,
+)
+@MustBeDocumented
+annotation class ExperimentalBackpackApi
```

Comment:
nice!

Reply:
can you mark all files in the package with this annotation? otherwise everything is red right now as it's using the experimental apis which makes it hard to parse the code